modules/custom/az_publication/config/install/views.view.az_publications.yml

```diff
@@ -659,7 +659,7 @@ display:
           exposed: false
           draggable_views_reference: 'az_publications_curate:curation_page'
           draggable_views_null_order: before
-          draggable_views_pass_arguments: true
+          draggable_views_pass_arguments: 1
```

Comment:
I made a new issue about this https://www.drupal.org/project/draggableviews/issues/3481312